etcdctlv3/command/get_command.go

```diff
@@ -40,8 +41,8 @@ func NewGetCommand() *cobra.Command {
 	cmd.Flags().StringVar(&getSortOrder, "order", "", "order of results; ASCEND or DESCEND")
 	cmd.Flags().StringVar(&getSortTarget, "sort-by", "", "sort target; CREATE, KEY, MODIFY, VALUE, or VERSION")
 	cmd.Flags().Int64Var(&getLimit, "limit", 0, "maximum number of results")
+	cmd.Flags().BoolVar(&getWithPrefix, "with-prefix", false, "enable keys with matching prefix")
```

Comment:
how about just `prefix`? (to be consistent with https://github.com/coreos/etcd/blob/master/etcdctlv3/command/watch_command.go#L43)